### PR TITLE
[Centcom] Nanotrasen Private Security 2.0

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR adds a lot of changes so I'll list them one by one, this is a continuation of my original NPS PR, however with criticism taken on: https://github.com/tgstation/tgstation/pull/24161

-------

Centcom Nanotrasen Security. The centcom docks now have two NT private security officers, though they don't seem very lifelike (AKA: Cardboard cutout).

privatecutout

------

New NT Private Sec cardboard cutout, can be acquired in game as you do with every other cardboard cutout. 

------

New passive, retaliate, NT Private Security NPCs. These NPCs are akin to their aggressive versions, however these officers will only attack someone who attacks them first. If attacked they will call for backup and engage the perpetrator with extreme prejudice.

They are also apart of the 'nanotrasanprivate' faction, meaning they will not attack each other, their aggressive counterparts or player controlled Human Private Sec.

Currently none spawn on any map.

--------

New cyropod Private Security Officer pods. These function akin to lavaland roles in that you click the sleeper and you become that role. The Cyro PSO is currently admin only, but spawns as apart of the same faction as the NPCs. They have all access to centcom.

alive

--------

If I were code competent I would suggest having a bunch of aggressive NPCs spawn in if a cult/gang takes over the escape shuttle, or a admin button to spawn in ten or so at the docks. But that is not the case.

Thank you for reading.

:cl: Steelpoint
add: Central Command has listened to complaints and, as such, has now stationed "real" Private Security Officers at centcom docks.
add: A new Nanotrasen Security Officer NPC variant is available to admins, this 'peaceful' version will only attack people who attack it first. Great for keeping order.
/:cl:

### CODE REVIEW

File: code/game/objects/items/cardboard_cutouts.dm [hunk 1]
```diff
@@ -168,6 +168,11 @@
 			desc = "A cardboard cutout of a laughter demon."
 			icon = 'icons/mob/mob.dmi'
 			icon_state = "bowmon"
+		if("Private Security Officer")
+			name = "Private Security Officer"
+			desc = "A cardboard cutout of a private security officer."
```
Comment (RemieRichards) [inline code/game/objects/items/cardboard_cutouts.dm]: "They seem to be ignoring you... Typical"
let's pretend they're not cardboard, but we'll all -know- they are.
you can put the desc on the mapped in only ones if you want.

Comment (Steelpoint) [inline code/game/objects/items/cardboard_cutouts.dm]: Good point

**Comment by Steelpoint** [comment]
The two map spawned cardboard cutouts have a new description: "They seem to be ignoring you... Typical."

Also, I've updated the Alive Private Security Officer to have general security access, so he can open the Centcom Sec Lockers.

alive

While amusing to think about, these cyro chambers don't spawn on Centcom.

**Comment by Jazaen** [comment]
Update the description, it says "nanotrasanprivate" instead of "nanotrasenprivate".

**Comment by LanCartwright** [comment]
Is this one specific individual or everyone they engage?

Also requesting a race based targeting mechanic please. It will be useful for winter storm project.

**Comment by Steelpoint** [comment]
They will only attack anyone who attacked them.

**Comment by Niknakflak** [comment]
Nobody likes the old PR because we would have to delete them all.  Now, they are cardboard on centcom, and pretty much nonexistent/mappers otherwise which is ok.  Maybe they'll never get used, who knows.  Meh

**Comment by PKPenguin321** [comment]
This is much better than before, might use those sleepers for ghosts at roundend if a baddy is stealing the shuttle

**Comment by Steelpoint** [comment]
Just as a note if you intend to spawn the cyro cells. There are only two main sec lockers that have a auto rifle in them each, as the officers spawn in un armed.

Though you can just spawn in a dozen rifles/lasers in the main room if you want.

**Comment by Armhulen** [comment]
Another good pr

**Comment by LanCartwright** [comment]
steelpoint, could I request race based interactions on these npcs please?

**Comment by Lumipharon** [comment]
Snpcs's or npc's?

The thought of Snpc sec officer that run around harm batoning anyone that violates space law amuses me greatly.

**Comment by Steelpoint** [comment]
Basic NPCs.

Also I have no idea how to do a race based interaction.

**Comment by LanCartwright** [comment]
By they engage in combat, do you mean all NPCs or just that one only?

Context being, officers A and B, assistant A attacks officer B, does officer A respond?

**Comment by Steelpoint** [comment]
In your scenario, officer A would also respond and they would both only attack assistant A.

In game, the retaliate officers will send out a 'distress call' if they are attacked, summoning all any other officers within 15 tiles to their location to assist.

**Comment by LanCartwright** [comment]
fukken sweet. I like this.

**Comment by LanCartwright** [comment]
What about dragging?

Could someone drag these guys away from their buddies?

**Comment by Steelpoint** [comment]
You can drag them like anything else and they won't mind though they will continue to walk around and likely fall out of your grasp.

Considering the only place these guys are likely to ever be spawned in is Centcom, I don't think we will ever have many issues of people abusing the 'help call' radius to move people to a hidden area to kill them.

**Comment by InsaneHyena** [comment]
>Ballistic weaponry
Fucking stop.

**Comment by Steelpoint** [comment]
I'm merely basing the NT Peacekeepers with the pre-existing NT Private Sec Officers, who use ballistics.

**Comment by InsaneHyena** [comment]
(((Pure coincidence)))

**Comment by ChangelingRain** [comment]
NT does not use ballistics. You are in error, and should correct this.

**Comment by Steelpoint** [comment]
>riot shotgun
>combat shotguns
>SABR smg
>ARG Assault Rifle
>Mataba

**Comment by InsaneHyena** [comment]
Didn't you add half of this shit yourself because you just can't contain your tacticool fetish?

**Comment by Steelpoint** [comment]
Err, I did not add any of those items at all.

**Comment by Steelpoint** [comment]
I also don't know how to make the NT Sec use lasers, besides they've used ballistics ever since they were implemented by whoever made them.

**Comment by optimumtact** [comment]
Yeah, they've always been that way

**Pull request merged**